src/parcsr_ls/par_mgr_setup.c

```diff
@@ -1925,3 +1968,88 @@ hypre_MGRSetupFrelaxVcycleData( void *mgr_vdata,
 
    return hypre_error_flag;
 }
+#if 0
```

Comment:
@oseikuffuor1 Do we want to keep this code?

Reply:
@victorapm this section has been removed. Thanks.